docs/API.md

```diff
@@ -728,6 +728,18 @@ const Counter = componentFromStream(props$ => {
 })
 ```
 
+### `mapPropsStreamWithConfig()`
```

Comment:
`setObservableConfig` is not so bad even it uses globals, in our daily work we use a lot of implicit configs at compile level - all that webpack, babel etc, so I'm not sure about deprecation.